blockchain/blockchain-wallet-v4-frontend
# Exchange state selection page

## Description
Adds state selection page for US residents trying to use Shapeshift

## Change Type
Feature

## Testing Steps
Create new wallet using VPN mocking US IP, go to exchange tab and see the state selection page, select whitelisted state and press continue. User should now be able to use exchange.  

Log out of wallet and back in and the state selection box should not reappear as data was persisted in kvStore.

## PR Creator Checklist

## PR Reviewer Checklist
